app/pages/DomainManager/index.js

```diff
@@ -22,9 +25,32 @@ class DomainManager extends Component {
     this.props.getMyNames();
   }
 
+  handleExportClick() {
+    let names = this.props.myDomains.map(({name}) => name);
+    let data = names.join("\n");
+
+    let savePath = dialog.showSaveDialogSync({
+      filters: [{ name: 'spreadsheet', extensions: ['csv'] }]
+    });
+
+    if(savePath) {
+      fs.writeFile(savePath, data, (err) => {
```

Comment:
you can actually create a csv in memory and trigger a download. Check out the method use here: https://github.com/kyokan/bob-wallet/blob/master/app/pages/Watching/index.js#L33-L41

Reply:
hmm, would you rather i use the watchlist code?

also, wdyt of centering the "download watchlist" button instead of the big blue extension button - which do you think looks better?

Reply:
yea let's use the watchlist code because it's the same functionality -- even if you just do copy-and-paste instead of refactoring into a utility function, it would be easier to maintain.

what do you mean by "_centering download watchlist button_"?